features/SPS007_Spatial-containment.feature

```diff
@@ -0,0 +1,23 @@
+@implementer-agreement
+@SPS
+@version1
+@E00040
+
+Feature: SPS007 - Spatial Containment
+The rule verifies that correct spatial containment - IfcRelContainedInSpatialStructure
+
+    Scenario: Subtypes of IfcElement are placed within the project spatial hierarchy
+        Given an IfcSpatialElement
+        Given A relationship IfcRelContainedInSpatialStructure from IfcSpatialElement to IfcElement and following that
+        Given is_a != IfcElementAssembly
```

Comment:
```suggestion
        Given Its Type != IfcElementAssembly
```

`is_a` is familiar for users of `ifcopenshell` but using `Its Type` as suggested - or `Its Entity Type` would be better.

Reply:
I would also prefer to not use `!=`  but `is not`.

Reply:
Implemented https://github.com/buildingSMART/ifc-gherkin-rules/pull/197/commits/db3e95c81359ad81abbd55232c0213e4a60d6400 

Both` !=` and `is not `are now accepted in the step_impl, but we are using `is not`

is_a is changed to allow` Its Type` or `Its Entity Type` in the gherkin statement for a type lookup of an entity. 
We're currently using` its type`

` Given Its type is not IfcElementAssembly`

Reply:
Looks good; thanks